ext/dav1d_android.sh

```diff
@@ -9,6 +9,8 @@
 # The git tag below is known to work, and will occasionally be updated. Feel
 # free to use a more recent commit.
 
+set -e
```

Comment:
You didn't add `set -e` to svt.sh. Is that intentional?

Reply:
> You didn't add `set -e` to svt.sh. Is that intentional?

No, let me fix that. I stopped at some point to try to determine why svt.sh and svt.cmd are present. I see the scripts they use from the SVT-AV1 tree are different.